README.md

```diff
@@ -7,3 +7,53 @@
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/prismarine-rng)
 
 Collection of utilities to work with minecraft RNG.
+
+## API
+
+## LCG
+?tbd
+
+## Random
+?tbd
+
+## crackPlayerSeed
+### crackPlayerSeed.crack(bits)
+Returns the decrypted player seed
+* `Returns` - Seed number
+* `bits` - Encrypted seed string?
+
+
+## enchantments
+### enchantments.findEnchantment(playerSeed, item, power, matching)
+* `Returns` - Object or null ?tbd
+* `playerSeed` - Number ?tbd
+* `item` - Item object ?tbd
+* `power` - ?tbd
+* `matching` - Matching function that returns a bool ?tbd
+
+### enchantments.getEnchantmentList(xpseed, item, slot, level)
+* `Returns` - Array ?tbd
+* `xpseed` - ?tbd
```

Comment:
the seed of the rng for the next enchant

Reply:
im assuming its the description for xpseed